HandEyeCalibration/Transform/Transform.cpp

```diff
@@ -0,0 +1,330 @@
+/*
+Example to show transformations to/from Transformation Matrix
+
+Zivid primarily operate with a (4x4) Transformation Matrix (Rotation Matrix + Translation Vector). 
+This example shows how to use Eigen to convert to and from:
+  AxisAngle, Rotation Vector, Roll-Pitch-Yaw, Quaternion
+
+ It provides convenience functions that can be reused in applicable applications.
+*/
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+#include <Eigen/Geometry>
+
+#include <opencv2/core/core.hpp>
+
+#include <iostream>
+
+enum class RotationConvention
+{
+    ZYX_Intrinsic,
+    XYZ_Extrinsic,
+    XYZ_Intrinsic,
+    ZYX_Extrinsic,
+    NOF_ROT
+};
+constexpr size_t nofRotationConventions = static_cast<size_t>(RotationConvention::NOF_ROT);
+
+struct RollPitchYaw
```

Comment:
Nice to pair the 3-element Euler angles with the rotation convention 👍 Would maybe call it `EulerAngles` instead of `RollPitchYaw`, as `RollPitchYaw` may give expectations to the order of rotations.

Reply:
I believe what @SatjaSivcev said is that this is a more recognized convention in the industry, and that even EulerAngles doesn't fully cover the variation

Reply:
I do not completely agree. As long as it is stated which conventions for Euler angles that is supported, it should be fine to call it `EulerAngles`. For `RollPitchYaw` one could for instance expect the `ZYX` convention.